frontend/src/js/controllers/challengeCtrl.js

```diff
@@ -961,7 +961,7 @@
         vm.downloadChallengeSubmissions = function() {
             if(vm.phaseId) {
                 var parameters = {};
-                parameters.url = "challenges/"+ vm.challengeId + "/phase/" + vm.phaseId + "/download_all_submissions_file/" + vm.fileSelected + "/";
+                parameters.url = "challenges/"+ vm.challengeId + "/phase/" + vm.phaseId + "/download_all_submissions/" + vm.fileSelected + "/";
```

Comment:
I am missing here the reason for this change. @RishabhJain2018 can you please point me out ?